DatadogInternal/Sources/Utils/SwiftExtensions.swift

```diff
@@ -115,3 +115,17 @@ extension Array {
         0 <= index && index < count ? self[index] : nil
     }
 }
+
+// MARK: - Bundle
+
+extension Bundle {
```

Comment:
/nit I think we should us the `DatadogExtended` pattern when defining `public` extension.